nim/tests/test_routes.py

```diff
@@ -0,0 +1,103 @@
+from nim.app import app
+import pytest
+import json
+
+
+@pytest.fixture
+def test_app():
```

Comment:
May be cleaner to call this `client` to avoid repeating `test_*` more than necessary.

Reply:
Changed 🙂